File: Algorithm/Backjun/bronze5/b2438/b2439.js
// 별 찍기 - 1
// https://www.acmicpc.net/problem/2438

const readline = require('readline')

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
})

// 별을 출력하는 함수
const printStar = (n) => {
  for (let i = 1; i <= n; i++) {
    console.log('*'.repeat(i))
  }
}

rl.question('', (input) => {
  const n = parseInt(input.trim())
  printStar(n)
  rl.close()
})

File: Algorithm/Backjun/bronze5/b2438/b2439.py
# 별 찍기 - 1
# https://www.acmicpc.net/problem/2438

# 별을 출력하는 함수
def printStar(n):
    for i in range(1, n+1):
        print("*" * i)

n = int(input())

printStar(n)
